src/main/java/net/pms/util/StringUtil.java

```diff
@@ -41,6 +41,18 @@
 	public static final String SEC_TIME_FORMAT = "%02d:%02d:%02.0f";
 	public static final String DURATION_TIME_FORMAT = "%02d:%02d:%05.2f";
 	public static final String NEWLINE_CHARACTER = System.getProperty("line.separator");
+	public static final long KIBI = 1L << 10;
```

Comment:
I do not agree with this unit choice.
From a long traditional/conventional use, memory capacities are reported in GB, MB...Informatic and technical literature mainly use that ones.
GiBi...are generally only used as speed/transfer unit. And for "commercial" publicity as it make it easy understandable for everyone.

https://www.reference.com/technology/ram-measured-6b59364967cdb1f4#
http://smallbusiness.chron.com/processor-speed-vs-memory-55329.html